vietnamese/TansStory/mystory.md

```diff
@@ -0,0 +1,2 @@
+Minh di dinh cu o My cung duoc 10 nam roi. Minh moi hoc ra truong.
+Minh dang kiem [viec lam](https://github.com/thuynh8/create-your-own-adventure/blob/master/vietnamese/TansStory/mystory.md) ma van khong co. Nhung minh se khong nan ma bo cuoc. Minh se co gang nhieu hon nua.
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.